api/src/main/resources/workbench-api.yaml

```diff
@@ -1419,6 +1419,28 @@ paths:
           description: User doesn't have the ACCESS_CONTROL_ADMIN authority
           schema:
             "$ref": "#/definitions/ErrorResponse"
+  "/v1/admin/users/batchSyncAccess":
+    post:
+      tags:
+        - userAdmin
+      consumes:
+        - application/json
+      description: 'Batch sync user access modules, asynchronously'
```

Comment:
Please leave a note that this mechanism should be deprecated once we have batch support from SAM (e.g. link CI ticket), and also that this should not normally be necessary - i.e. this his handled automatically by batchsyncuseraccess